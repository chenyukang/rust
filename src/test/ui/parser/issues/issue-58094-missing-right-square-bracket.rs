// Fixed in #66054.
// ignore-tidy-trailing-newlines
// error-pattern: aborting due to previous error
#[Ð… //~ ERROR unclosed delimiter