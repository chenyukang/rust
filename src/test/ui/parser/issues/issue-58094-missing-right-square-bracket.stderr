error: this file contains an unclosed delimiter
  --> $DIR/issue-58094-missing-right-square-bracket.rs:4:33
   |
LL | #[Ð…
   |  - unclosed delimiter           ^

error: aborting due to previous error

